apps/dashboard/app/apps/dev_router.rb

```diff
@@ -1,12 +1,10 @@
 # The router class for all development apps.
 class DevRouter
-  attr_reader :name, :owner, :caption, :category
+  attr_reader :name, :owner
 
   def initialize(name, owner=OodSupport::Process.user.name)
     @name = name.to_s
     @owner = owner
-    @caption = I18n.t('dashboard.development_apps_caption')
-    @category = "Sandbox Apps"
```

Comment:
I see why you made these changes, but thinking about it, I think I prefer the `attr_reader` way. Which is to say, I think I prefer the way this was originally. Maybe we should refactor `sys_router` to confirm to this pattern instead of the other way around?

I think I prefer this pattern because using `attr_reader`s seems more idiomatic to me, though that could just be an opinion. The other reason I think is performance. `@caption` being assigned in the initializer means we only have to do that hash lookup once. Subsequent calls to `caption` will just return the value, no need to look it up.

The way you have it now, it has to make lookups all the time - even though the value is constant at this point.